### PR TITLE
Scenarios can be modelled

### PR DESCRIPTION
**Pull request by manne**
screenshot
image

### CODE REVIEW

**Comment by manne** [comment]
the frontend github link is correctly built 😎 <https://github.com/AITGmbH/tapiodeveloperapp/tree/34ae45df55eff0ee4e404fd90697fb75ca198656/src/web/src/app/scenario-sample>

File: src/web/Controllers/ScenarioController.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+using Aitgmbh.Tapio.Developerapp.Web.Services;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Aitgmbh.Tapio.Developerapp.Web.Controllers.Scenario
+{
+    [Produces("application/json")]
+    [Route("api/[controller]")]
+    public class ScenarioController : Controller
+    {
+        private readonly IScenarioCrawler _scenarioCrawler;
```
Comment (SebastianRoether) [inline src/web/Controllers/ScenarioController.cs]: Benutz mal hier auch das ScenarioRepository

File: src/web/Startup.cs [hunk 2]
```diff
@@ -27,6 +30,9 @@ public void ConfigureServices(IServiceCollection services)
             services
                 .AddMvc()
                 .SetCompatibilityVersion(CompatibilityVersion.Version_2_2);
+            services
```
Comment (SebastianRoether) [inline src/web/Startup.cs]: can't you just remove this line?

File: src/web/Controllers/ScenarioController.cs (hunk 1 above)
Comment (manne) [reply src/web/Controllers/ScenarioController.cs]: for sure

File: src/web/Startup.cs (hunk 2 above)
Comment (manne) [reply src/web/Startup.cs]: i switched the order

**Pull request merged**